eva/optimizer/group.py

```diff
@@ -49,6 +50,10 @@ def __init__(self, group_id: int):
     def group_id(self):
         return self._group_id
 
+    @property
+    def aliases(self):
+        return self._aliases
```

Comment:
aliases are more like output objects of the operators (given operators are logically equivalent, the all should have the same output objects) in the group?

Reply:
What do you mean?

Reply:
@xzdandy do you mean that we need to extract it from whichever operator has the TableRef?

Reply:
I think that after binder, every operator should have output objects binded (called aliases?). Correct me if I am wrong.

Reply:
I should change it from list to set to avoid keeping duplicates

Reply:
Does that answer the concern?